cinema/views.py

```diff
@@ -5,11 +5,21 @@
 from rest_framework.authentication import TokenAuthentication
```

Comment:
do you need this import? you should use jwt instead. remove all imports you don't use